### ISSUE TITLE
Link to admin panel

### ISSUE THREAD

**Issue by karlabbott** [vector-im/element-web#21961]
#### What would you like to do?

Similar to the "Upgrade to Element Home" button that appears for matrix.org users, we would like to have a button that allows a customer direct access to their admin panel. If their homeserver is hosted in EMS, then the link would be to the EMS admin page. If their homeserver is hosted on-premise, then the link would be to the admin panel served up by the synapse-admin package.

#### Why would you like to do it?

We would like to do this to give our customers quicker access to their admin panels.

#### How would you like to achieve it?

By putting a button in the user menu in the same spot as the "Upgrade to Element Home" button.

### Have you considered any alternatives?

The alternative is what exists today where our customers have to keep track of these URLs independent of our application.

**Comment by daniellekirkwood** [comment]
<img width="449" alt="Screenshot 2022-04-27 at 10 51 36" src="https://user-images.githubusercontent.com/89144281/165831658-f050c447-c2db-4ad9-bcfe-f4f89172a07f.png">

Similar to this ☝️

**Comment by t3chguy** [comment]
@novocaine that would be a trivial thing to add but wouldn't help EMS users on Element Desktop since EMS does not provide custom Desktop builds at this time

**Comment by nadonomy** [comment]
@karlabbott if you end up actioning this and need design support let me know, there'd be some minor considerations around where to place it (e.g. the iconography if it's in this context menu, if there's a more natural place to expose it in settings, etc).

**Comment by novocaine** [comment]
> @novocaine that would be a trivial thing to add but wouldn't help EMS users on Element Desktop since EMS does not provide custom Desktop builds at this time

right; I guess the config could be a set of HS pattern matching rules and according admin links e.g. 

```
[
  ["*.element.io", "https://element.io/admin"]
]
```

**Comment by t3chguy** [comment]
It'd be great to avoid such, driving via a .well-known config like we do for jitsi etc would be far more scalable and agnostic

**Comment by novocaine** [comment]
> It'd be great to avoid such, driving via a .well-known config like we do for jitsi etc would be far more scalable and agnostic

agree that sounds better

**Comment by nadonomy** [comment]
> @nadonomy -- It would be brill if someone from the design team could help to propose how this would look in the clients? Once we have that we can work out who would be best placed to implement the designs.

@rxl881 on it!
